deeppavlov/models/kbqa/entity_linking.py

```diff
@@ -13,317 +13,312 @@
 # limitations under the License.
 
 import re
-import sqlite3
 from logging import getLogger
 from typing import List, Dict, Tuple, Optional, Any
-from collections import defaultdict
+from collections import defaultdict, Counter
 
+import numpy as np
 import nltk
```

Comment:
redundant

Reply:
I mixed up nltk with numpy and wrongly deleted numpy in one of commits